app/blueprints/marketing.py

```
@@ -0,0 +1,390 @@
+        download = LeadMagnetDownload(
+            lead_magnet_id=magnet.id,
+            email=email,
+            ip_address=request.remote_addr,
+            user_agent=request.user_agent.string[:255]
+        )
+        magnet.download_count = (magnet.download_count or 0) + 1
+        db.session.add(download)
+        db.session.add(magnet)
+        db.session.commit()
+        logger.info("Lead magnet downloaded: %s by %s", magnet.name, email)
+        return True
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)